### PR TITLE
Advanced Technology Labs (ATL) Mantis EDU support

### PR DESCRIPTION
**Pull request by dagar**
Screen Shot 2021-07-20 at 5 06 37 PM

### CODE REVIEW

File: boards/yuneec/mantis-edu/src/board_config.h [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+ */
+#define BOARD_ADC_USB_CONNECTED (px4_arch_gpioread(GPIO_OTGFS_VBUS))
+#define BOARD_ADC_USB_VALID     BOARD_ADC_USB_CONNECTED
+
+/* FMUv5 never powers odd the Servo rail */
+#define BOARD_ADC_SERVO_VALID     (1)
+
+#define BOARD_ADC_BRICK1_VALID  (!px4_arch_gpioread(GPIO_nVDD_BRICK1_VALID))
+#define BOARD_ADC_BRICK2_VALID  (0)
+
```
Comment (Abner0907) [inline boards/yuneec/mantis-edu/src/board_config.h:166]: you have defined 
```
#define BOARD_ADC_BRICK_VALID           1
```
So I think this is no needed

File: .ci/Jenkinsfile-compile [hunk 2]
```diff
@@ -99,7 +99,8 @@ pipeline {
                      "px4_fmu-v6x_default",
                      "px4_io-v2_default",
                      "spracing_h7extreme_default",
-                     "uvify_core_default"
+                     "uvify_core_default",
+                     "yuneec_mantis-edu_default"
```
Comment (julianoes) [inline .ci/Jenkinsfile-compile:103]: We'll have to rename `yuneec` to `atl` everywhere.

Comment (dagar) [reply .ci/Jenkinsfile-compile:103]: Done.

**Comment by dagar** [comment]
TODO
 - RC mapping
 - baro bug driver

File: boards/atl/mantis-edu/nuttx-config/nsh/defconfig [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+CONFIG_BOARD_RESET_ON_ASSERT=2
+CONFIG_BUILTIN=y
+CONFIG_C99_BOOL8=y
+CONFIG_CDCACM=y
+CONFIG_CDCACM_PRODUCTID=0x0061
+CONFIG_CDCACM_PRODUCTSTR="PX4 ATL Mantis-EDU"
+CONFIG_CDCACM_RXBUFSIZE=600
+CONFIG_CDCACM_TXBUFSIZE=12000
+CONFIG_CDCACM_VENDORID=0x26ac
+CONFIG_CDCACM_VENDORSTR="ATL"
```
Comment (julianoes) [inline boards/atl/mantis-edu/nuttx-config/nsh/defconfig:53]: This is `Advanced Technology Labs` in the bootloader.

Comment (dagar) [reply boards/atl/mantis-edu/nuttx-config/nsh/defconfig:53]: Thanks

**Comment by dagar** [comment]
#### TODO (likely in a followup PR)
 - gimbal manual control
 - review tunes (power off, etc)
 - optical flow
 - distance sensor (metadata wrong, land detector improvements)
 - logging options (why does starting log streaming fail?)

**Pull request merged**